bert/src/bert_layers.py

```diff
@@ -101,7 +101,7 @@ def forward(
         inputs_embeds: Optional[torch.FloatTensor] = None,
         past_key_values_length: int = 0,
     ) -> torch.Tensor:
-        if (input_ids is not None) != (inputs_embeds is not None):
+        if (input_ids is not None) == (inputs_embeds is not None):
```

Comment:
I think `inputs_embeds` is usually left as `None`, so I'm wondering why I haven't seen this exception being raised... In any case, your fix looks correct. Assuming that the tester script runs successfully (confirm by running `composer main.py yamls/test/main.yaml`), then LGTM.

Reply:
Got it. Thanks for catching this!